### ISSUE TITLE
disable export file

### ISSUE THREAD

**Issue by priamai** [os-threat/Stix-ORM#1]
We should disable : export_final.json being produced.

**Comment by brettforbes** [comment]
no, it is useful for debugging purposes, it is just the location that needs to be changed, back into the ORM, I have done this

**Comment by priamai** [comment]
idea! what if add a logging level parameter, so we can then disable/enable the verbosity and such file?
